File: PalindromeNumber.py
def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        convert = str(x)
        xBack = len(convert) - 1
        for item in range(0, len(convert)/2):
            if convert[item] != convert[xBack]:
                return False;
            xBack -= 1;
            
            
        return True; 

File: c++/palindromeNumber.cpp
 bool isPalindrome(int x) {
        string convert = std::to_string(x);
        int xBack = convert.length() - 1;
        for (int i = 0; i < convert.length()/2; ++i){
            if(convert[i] != convert[xBack]){
                return false;
            }
            xBack -= 1;
        }
        
        return true; 
    }
